lib/src/code_generator/func.dart

```diff
@@ -117,7 +117,11 @@ class Func extends LookUpBinding {
     if (dartDoc != null) {
       s.write(makeDartDoc(dartDoc));
     }
-
+    // Resolve name conflicts in function parameter names.
```

Comment:
5 line fix, I love it! Shows our code is written in a good way!

Reply:
❣️